src/unix/os390-syscalls.c

```diff
@@ -294,10 +297,16 @@ int epoll_wait(uv__os390_epoll* lst, struct epoll_event* events,
 
     ev.fd = pfds[i].fd;
     ev.events = pfds[i].revents;
-    events[reventcount++] = ev;
+    if (pfds[i].revents & POLLIN && pfds[i].revents & POLLOUT)
+      reventcount += 2;
+    else if (pfds[i].revents & (POLLIN | POLLOUT))
+      ++reventcount;
```

Comment:
Maybe this?
```c
reventcount += !!(pfds[i].revents & POLLIN);
reventcount += !!(pfds[i].revents & POLLOUT);
```
And since you're referencing `pfds[i]` a lot I'd maybe assign it to a `struct pollfd *pfd`.